recipes/xeus-octave/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "xeus_octave" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapgenic/xeus-octave/archive/{{ version }}.tar.gz
+  sha256: 0320fddf6b8e1ae4d23a5489e6b1d609d37bab7b2641fa16c6ae92ace5c0e759
+  patches:
+    - 0001-Improve-build-system.-Add-GLFW-osmesa-support.patch
+    - 0002-Update-display-system-to-support-multiple-mimes.patch
+    - 0003-Update-README-notebook.patch
+    - 0004-Fix-for-osx.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pkg-config  # [osx]
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+  host:
+    - nlohmann_json >=3.9.1,<4.0
+    - octave
+    - xeus >=1.0,<2.0
+    - xproperty
+    - xorg-libxfixes  # [linux]
+    - glfw
+    - libpng
+  run:
+    - glfw
+    - octave
+
+test:
+  requires:
+    - jupyter_kernel_test
+  commands:
+    - echo ${PREFIX}
+    - ls ${PREFIX}
+    - ls ${PREFIX}/bin/
+    - test -f ${PREFIX}/bin/xeus-octave  # [unix]
+
+about:
+  home: https://github.com/rapgenic/xeus-octave
+  license: GPL
```

Comment:
The linter isn't happy with this, please find out the correct one according to SPDX. You can remove `license_familiy` afterwards, that will be autofilled if this one is correct.